pkg/rtc/mediatrackreceiver.go

```diff
@@ -780,7 +781,9 @@ func (t *MediaTrackReceiver) UpdateAudioTrack(update *livekit.UpdateLocalAudioTr
 	t.lock.Lock()
 	trackInfo := t.TrackInfo()
 	clonedInfo := utils.CloneProto(trackInfo)
-	clonedInfo.AudioFeatures = update.Features
+
+	clonedInfo.AudioFeatures = sutils.DedupeSlice[livekit.AudioTrackFeature](update.Features)
```

Comment:
it can't infer the type from the argument?

Reply:
let me try.

Reply:
It does, something was failing compile which made me add it, but don't remember what it is now :-(